.github/workflows/ci.yml

```
@@ -115,72 +112,33 @@ jobs:
       - name: Prepare
         id: prep
         run: |
-          if [[ "${{ github.event_name }}" == "pull_request" ]]; then
-            VERSION="snapshot-$(echo ${GITHUB_HEAD_REF} | sed 's#[^a-zA-Z0-9_-]#-#g')"
-          else
-            if [[ "${GITHUB_REF_TYPE}" == "tag" ]]; then
-              VERSION="${GITHUB_REF_NAME}"
-            elif [[ "${GITHUB_REF_NAME}" == "master" ]]; then
-              VERSION="snapshot"
-            else
-              VERSION="snapshot-$(echo ${GITHUB_REF_NAME} | sed 's#[^a-zA-Z0-9_-]#-#g')"
-            fi
-          fi
-
-          LABELS=""
-          if [[ "${GITHUB_REF_TYPE}" != "tag" ]] && [[ ! "${GITHUB_REF_NAME}" =~ ^release-* ]]; then
-            LABELS="quay.expires-after=10d"
-          fi
-
-          build_date="$(date -u +"%Y-%m-%dT%H:%M:%S+00:00")"
-          go_build_args=(
-            "-ldflags=-X 'github.com/Dynatrace/dynatrace-operator/src/version.Version=${VERSION}'"
-            "-X 'github.com/Dynatrace/dynatrace-operator/src/version.Commit=${GITHUB_SHA}'"
-            "-X 'github.com/Dynatrace/dynatrace-operator/src/version.BuildDate=${build_date}'"
-            "-s -w"
-          )
-
-          # Set output parameters.
-          echo ::set-output name=goargs::"${go_build_args[@]}"
-
-          echo ::set-output name=labels::${LABELS}
-          echo ::set-output name=version::${VERSION}
-
-          # Get licenses if no cache exists
-          if ! [ -d ./third_party_licenses ]; then
-            go get github.com/google/go-licenses && go-licenses save ./... --save_path third_party_licenses --force
-          fi
-
-          # Get dependencies
-          go get -d ./...
-          ls -la $HOME/go/pkg/mod
-          cp -r $HOME/go/pkg/mod .
+          hack/build/ci/prepare_variables_and_download_go_build_deps.sh
```

```suggestion
          hack/build/ci/prepare-build-environment.sh
```

Generally I am more a fan of dashes instead of underscores, because if text is selected with Ctrl+Shift the selection is more granular since it stops at dashes. However, that's just a personal preference